CODEOWNERS

```diff
@@ -4,6 +4,10 @@
 /oneTBB/src/tbb/ @dnmokhov
 /oneTBB/src/tbb/ @JhaShweta1
 /oneTBB/src/tbb/ @sarathnandu
+/oneTBB/include/oneapi/tbb/parallel_* @pavelkumbrasev
+/oneTBB/include/oneapi/tbb/concurrent_* @kboyarinov
+/oneTBB/include/oneapi/tbb/flow_graph* @kboyarinov
```

Comment:
Should internal headers also be added here?
As far as I understand, this list would be used by GitHub to better suggest reviewers for the files and if only internals would be affected, it may also be useful.

Reply:
I presume you're talking about the headers in folder "detail" - Updated that as well.